src/cdk-experimental/popover-edit/edit-ref.ts

```diff
@@ -57,6 +63,7 @@ export class EditRef<FormValue> implements OnDestroy {
   }
 
   ngOnDestroy(): void {
+    this._editEventDispatcher.editRef = null;
```

Comment:
This has the potential of clearing a different edit ref. Maybe there should be a `if (this._editDispatcher.editRef === this)` around it?